folder/git_tutorial.py

```diff
@@ -3,5 +3,12 @@ def add_number(a,b,c):
     return sum([a,b,c])
 
 
+#1. extend add_numbers to allow an arbitrary number of parameters 
+#2. investigate and resolve security exploit of add_numbers()
+#3. make the implementation of add_numbers() more performant
+#4. add documentation for add_numbers()
```

Comment:
i find the section vague . Please consider adding a verbal description of what these commands are actually doing .